athletics.lic

```diff
@@ -2,22 +2,26 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#athletics
```

Comment:
This script isn't integrated with `crossing-training`. Do you plan to also update the athletics logic in `crossing-training` to support the climbing rope?